packages/components/nodes/chatmodels/ChatOpenAI/ChatOpenAI.ts

```diff
@@ -19,7 +19,7 @@ class ChatOpenAI_ChatModels implements INode {
     constructor() {
```

Comment:
is it possible that we override the method `_generate` method and just append the images to the `messages` param in the func